### ISSUE TITLE
Content Control Wrapper AccessAdminPanel permission

### ISSUE THREAD

**Issue by ziic** [OrchardCMS/Orchard#7589]
Hello.
Shape template **Content.ControlWrapper.cshtml** by default checks only _Permissions.EditContent_ permission to display "Edit" button and link goes to admin control panel. But maybe it will be more useful to check also _StandardPermissions.AccessAdminPanel_ permission? I think It does not make sense to show button that leads to admin panel if user cannot use the panel.

**Comment by sebastienros** [comment]
Does it make sense for a user to have `EditContent` permission without `AccessAdminPanel` ? 

Is it already possible to remove `EditContent` on the same role that you removed `AccessAdminPanel` from?

Otherwise we can also check both permission.

**Comment by ziic** [comment]
Yes, my opinion is to check both permissions. I constantly override this shape-template im my projects (where the feature is used) and check both permissions. I understand that the issue is not important at all and it is easy to fix by custom template, but maybe such verification should be out of the box
